src/lib/components/mods/ModGrid.svelte

```diff
@@ -45,9 +45,19 @@
   let totalMods: number;
 
   let searchField = search;
+  let searchDisabled = true;
+  let searchButtonClass = 'variant-filled-primary';
+  const showPagination: boolean = $mods && !$mods.error && !$mods.fetching;
 
   let timer: number;
   $: {
+    if (searchField.length > 2) {
+      searchDisabled = false;
+      searchButtonClass = 'variant-filled-primary';
+    } else {
+      searchDisabled = true;
+      searchButtonClass = 'variant-filled-surface';
+    }
```

Comment:
This can be simplified to a reactive definition for the two variables, with a ternary operator.